src/physics/em/detail/BetheBlochInteractor.hh

```diff
@@ -0,0 +1,89 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file BetheBlochInteractor.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "base/Macros.hh"
+#include "base/Types.hh"
+#include "physics/base/Interaction.hh"
+#include "physics/base/ParticleTrackView.hh"
+#include "base/StackAllocator.hh"
+#include "physics/base/Secondary.hh"
+#include "physics/base/Units.hh"
+#include "physics/material/ElementView.hh"
+#include "physics/material/MaterialView.hh"
+#include "BetheBloch.hh"
+
+namespace celeritas
+{
+namespace detail
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Muon bremsstrahlung.
+ *
+ * \note This performs the same sampling routine as in Geant4's
+ * G4MuBremsstrahlungModel class, as documented in section 11.2 
+ * of the Geant4 Physics Reference (release 10.6).
```

Comment:
Name and sections need to be updated.

Reply:
I renamed the classes to MuBremsstrahlung, so I think the name and section match the code now.

Reply:
Yes, that looks good now. Thanks.